### PR TITLE
(maint) Restore plural messages

### PR DESCRIPTION
**Pull request by MikaelSmith**
Until Leatherman adds plural i18n helpers, we don't want a release to
backtrack on pluralizing message strings. Restore basic pluralization in
a way similar to what the i18n helpers will do.

### CODE REVIEW

**Comment by MikaelSmith** [comment]
@kylog

File: lib/src/results_storage.cc [hunk 1]
```diff
@@ -262,8 +262,13 @@ unsigned int ResultsStorage::purge(
         });
 
     // TODO(ale): deal with locale & plural (PCP-257)
-    LOG_INFO("Removed {1} directories from '{2}'",
-             num_purged_dirs, spool_dir_path_.string());
+    if (num_purged_dirs == 1) {
+        LOG_INFO("Removed {1} directorie from '{2}'",
```
Comment (kylog) [inline lib/src/results_storage.cc]: directory

**Comment by kylog** [comment]
LGTM, other than the one typo'd singular.

File: lib/src/results_storage.cc (hunk 1 above)
Comment (MikaelSmith) [inline lib/src/results_storage.cc]: Fixed.

**Comment by parisiale** [comment]
👍

**Pull request merged**